data/sql/updates/pending_db_world/ru_An _Aggressive_Defense_offer.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO quest_offer_reward_locale (ID, locale, RewardText, VerifiedBuild) VALUES (1025, 'ruRU', 'Славная работа, $N. Спасибо тебе.', 18019);
```

Comment:
```suggestion
DELETE FROM `quest_offer_reward_locale` WHERE `ID`=1045 AND `locale`='ruRU';
INSERT INTO `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`, `VerifiedBuild`) VALUES (1045, 'ruRU', 'Славная работа, $N. Спасибо тебе.', 18019);
```

Reply:
so just more of a explination, we do a delete first before a insert to ensure re-runability

Reply:
> Yes I know. It’s just that the database didn’t have this entry at all, and I thought that it could be done that way.